src/model/UserRegistry.java

```diff
@@ -0,0 +1,26 @@
+package model;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class UserRegistry {
+
+  private static Set<User> userRegistry = new HashSet<>();
+
```

Comment:
hashmap would have made more sense.

Reply:
thanks for reviewing, please could you help me understand what types to use in a hashmap, I mean, this was meant to be a registry/ tmp-storage of all users. If you could let me know what you were thinking about hashmap, it would be great.